diff --git a/inst/lib/bs-a11y-p/src/js/dropdown.js b/inst/lib/bs-a11y-p/src/js/dropdown.js
index b1ef2d4..1f4c021 100644
--- a/inst/lib/bs-a11y-p/src/js/dropdown.js
+++ b/inst/lib/bs-a11y-p/src/js/dropdown.js
@@ -46,7 +46,7 @@
         }
         setTimeout(function() {
           if(!$.contains(that, document.activeElement)){
-            $this.parent().find('[data-toggle=dropdown]').dropdown('toggle')
+            $this.parent().find('[data-toggle=dropdown]').dropdown('hide');
           }
         }, 150)
        })
