diff --git a/inst/lib/bs-colorpicker/js/bootstrap-colorpicker.js b/inst/lib/bs-colorpicker/js/bootstrap-colorpicker.js
index 4df8432..739b077 100644
--- a/inst/lib/bs-colorpicker/js/bootstrap-colorpicker.js
+++ b/inst/lib/bs-colorpicker/js/bootstrap-colorpicker.js
@@ -4164,7 +4164,9 @@ var PopupHandler = function () {
 
       this.popoverTarget.popover(_jquery2.default.extend(true, {}, _options2.default.popover, cp.options.popover, { trigger: 'manual', content: cp.picker, html: true }));
 
-      this.popoverTip = (0, _jquery2.default)(this.popoverTarget.popover('getTipElement').data('bs.popover').tip);
+      this.popoverTip = (window.bootstrap && window.bootstrap.Popover && window.bootstrap.Popover.getInstance) ? 
+        (0, _jquery2.default)(window.bootstrap.Popover.getInstance(this.popoverTarget[0]).getTipElement()) :
+        (0, _jquery2.default)(this.popoverTarget.popover('getTipElement').data('bs.popover').tip);
       this.popoverTip.addClass('colorpicker-bs-popover');
 
       this.popoverTarget.on('shown.bs.popover', _jquery2.default.proxy(this.fireShow, this));
